packages/plugin/tests/__snapshots__/parser.spec.ts.snap

```diff
@@ -8,10 +8,6 @@ Object {
         Object {
           description: Object {
             block: true,
-            gqlLocation: Object {
```

Comment:
I don't see a reason to have `gqlLocation` field, current is unused, also this info can be retrieved with `range` field